### DISCUSSION TITLE
Is it possible to call any mappable action via command-line instead of mapping?

### DISCUSSION THREAD

**Discussion by EgZvor** [kovidgoyal/kitty#8005] [Q&A] [answered]
I want to extract kitty's scrollback and use it in another command, but using a command line, not a mapping.

More specifically, I want to edit command-line via shell and additionally pass it a scrollback.

**Comment by kovidgoyal** [comment, marked as answer]
You can use remote control to run any mappable action via kitten @
action

**Reply by EgZvor** [reply]
Is it possible to stay in the current window/shell as well? `launch --type` has only `overlay` but I can't capture the output from `cat`, for example, in the current shell.

**Reply by EgZvor** [reply]
OK, I found a way to do this. `kitten @ get-text --extent all` does what I wanted.

**Comment by EgZvor** [comment]
Here's the result! https://gist.github.com/EgZvor/0a6a1f4252e74f322073b07a5f15af94
